docker-compose.yml

```diff
@@ -78,6 +80,25 @@ services:
       - agile-core:agile
     restart: always
 
+
+
+  agile-idm:
+    image: agileiot/agile-idm-armv7l
```

Comment:
Yes, well you'd have to add it for each repo. There's an example here: https://github.com/Agile-IoT/agile-example

If you don't want to use versionist, travis etc. Then you should manually version it, by tagging each docker image before pushing.